csrankings-s.csv

```diff
@@ -1200,6 +1200,7 @@ Shriram Krishnamurthi,Brown University,https://cs.brown.edu/~sk,Cxwj5hEAAAAJ
 Shrisha Rao 0001,IIIT Bangalore,http://www.iiitb.ac.in/faculty_page.php?name=ShrishaRao,SxYeZQcAAAAJ
 Shruti Vyas,University of Central Florida,https://www.cs.ucf.edu/person/shruti-vyas/,15YqUQUAAAAJ
 Shu Kong,University of Macau,https://www.fst.um.edu.mo/people/skong/,sm9FdLoAAAAJ
+Shu Yang 0002,Shenzhen University,https://ieeexplore.ieee.org/author/37086450194,JKR65R0AAAAJ

```

Comment:
Associate Researcher, not TT faculty. Remove.

Reply:
Sorry the last faculty member Jiajun Wen in this request  should be assistant professor but I made mistakes to spell as "associated professor" and closed the conversation so I can not make any comments. 

For faculty member Shu Yang, his official position is also an assistant professor. Below is the screenshot from the official website homepage of College of Computer Science and Software Engineering in Shenzhen University. Associated Researcher is the outdated title. 
![Shu Yang](https://github.com/user-attachments/assets/66836561-ec57-4989-9cd1-939c19e0cbc8)
[https://csse.szu.edu.cn/en/pages/teacherTeam/index?p=1&s=Shu%20Yang](url)